networkx/algorithms/time_dependent.py

```diff
@@ -61,7 +63,11 @@ def cd_index(G, node, *, time_delta=timedelta(days=5 * 365), time="time", weight
     >>> G.add_nodes_from([(n, nodes[n]) for n in nodes])
     >>> edges = [(1, 3), (1, 4), (2, 3), (3, 4), (3, 5)]
     >>> G.add_edges_from(edges)
-    >>> cd = nx.cd_index(G, 3, time="time", weight="weight")
+    >>> delta = timedelta(days=5 * 365)
+    >>> nx.cd_index(G, 3, time_delta=delta, time="time")
+    0.5
+    >>> nx.cd_index(G, 3, time_delta=delta, time="time", weight="weight")
+    0.12
```

Comment:
I'm wondering if we could add an example that doesn't use datetime. I tried quickly to take this example and make the dates just a year as an int e.g. '2012'.  To get it to work, I had to make `time_delta=4` instead of `time_delta=5`. Did I miss something? Or maybe there is an "off-by-one" error in the 5*365 value (or in how time_delta is used).

```suggestion
    0.12
    
    Integers can also be used for the time values:
    >>> node_times = {1: 2015, 2: 2012, 3: 2010, 4: 2008, 5: 2014}
    >>> nx.set_node_attributes(G, node_times, "new_time")
    >>> nx.cd_index(G, 3, time_delta=4, time="new_time")
    0.5
    >>> nx.cd_index(G, 3, time_delta=4, time="new_time")
    0.12
```